moveit_ros/hybrid_planning/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(moveit_hybrid_planning)
+
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(rclcpp_action REQUIRED)
+find_package(moveit_msgs REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_ros_planning_interface REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(std_srvs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(trajectory_msgs REQUIRED)
+find_package(control_toolbox REQUIRED)
```

Comment:
Everything else is nicely alphabetized. You might as well alphabetize this, too.

(Also, check that control_toolbox is actually needed. I don't see it used anywhere)